cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -210,17 +210,20 @@ Created
 _______
 +--------------+-----------------------+
 | Operator     | - =                   |
-|              | - ">"                 |
+|              | - <>                  |
```

Comment:
really not sure at all about the operator '<>' for different, we can discuss IRL about it